### PR TITLE
Updating HAProxy performance numbers for 3.10.

### PR DESCRIPTION
**Pull request by jmencak**
This commit updates the HAProxy performance for 3.10 and adds HTTP close (non-keepalive) scenarios for the 4 route terminations.  Also updating the factors that affect router performance.

### CODE REVIEW

**Comment by jmencak** [comment]
@ahardin-rh PTAL

**Comment by jmencak** [comment]
All great suggestions, thanks Jeremy.  Updated.

File: scaling_performance/routing_optimization.adoc [hunk 1]
```diff
@@ -22,32 +22,54 @@ xref:../install_config/router/index.adoc#install-config-router-overview[router]
...
+* number of target routes 
+
+* backend server page size
+
+* underlying infrastructure (network/SDN solution, CPU...)
+
+While performance in your specific environment will vary, our lab
+tests on a public cloud instance of size 4 vCPU/16GB RAM, a single
+HAProxy router handling 100 routes terminated by backends serving
+1kB static pages is able to handle
```
Comment (ahardin-rh) [inline scaling_performance/routing_optimization.adoc]: I would format this section a little differently.

.. 1KB static page is able to handle the following.

In HTTP keep-alive mode scenarios:
<table>

In HTTP close (no keep-alive) scenarios:
<table>

File: scaling_performance/routing_optimization.adoc [hunk 2]
```diff
@@ -22,32 +22,54 @@ xref:../install_config/router/index.adoc#install-config-router-overview[router]
...
+
+* TLS session resumption client support
+
+* number of concurrent connections per target route
+
+* number of target routes 
+
+* backend server page size
+
+* underlying infrastructure (network/SDN solution, CPU...)
```
Comment (ahardin-rh) [inline scaling_performance/routing_optimization.adoc]: , CPU, and so on)

File: scaling_performance/routing_optimization.adoc [hunk 3]
```diff
@@ -22,32 +22,54 @@ xref:../install_config/router/index.adoc#install-config-router-overview[router]
...
-see results in the lower range. With HTTP keep-alive, a single HAProxy router is
-capable of saturating 1 Gbit NIC at page sizes as small as 8 kB.
-
-The table below shows HTTP keep-alive performance on such a public cloud
-instance with a single HAProxy and 100 routes:
-
-[cols="2,3,3",options="header"]
+When evaluating a single HAProxy router performance in terms of
+HTTP requests handled per second, the performance varies depending
+on many factors.  In particular:
```
Comment (ahardin-rh) [inline scaling_performance/routing_optimization.adoc]: There should only be one space between sentences

**Comment by ahardin-rh** [comment]
@mffiedler Are you able to provide QE review? Thanks!

File: scaling_performance/routing_optimization.adoc (hunk 3 above)
Comment (jeremyeder) [reply scaling_performance/routing_optimization.adoc]: guess we cant be friends anymore

**Comment by mffiedler** [comment]
@schituku Please review and comment as needed.  Thanks.

File: scaling_performance/routing_optimization.adoc (hunk 3 above)
Comment (ahardin-rh) [reply scaling_performance/routing_optimization.adoc]: @jeremyeder :sob:

**Comment by schituku** [comment]
@mffiedler @ahardin-rh /lgtm

**Comment by ahardin-rh** [comment]
Thanks!

**Pull request merged**